### PR TITLE
User can choose space

### PR DESCRIPTION
**Pull request by ericdrosado**
screen shot 2017-11-30 at 4 26 51 pm

### CODE REVIEW

File: Tests/BoardTest.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        private Board board;
+        
+        public BoardTest() {
+            this.board = new Board();
+        }
+
+        [Theory]
+        [InlineData(0)]
+        [InlineData(4)]
+        public void AddMarkerToGameBoard(int move) {
```
Comment (cdemyanovich) [inline Tests/BoardTest.cs]: Even though there's only two cases, I like that you're aware of opportunities to use `Theory`.

File: Tests/UITest.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+namespace Tests.TicTacToe {
+
+    public class UITest {
+
+        private UI ui;
+
+        public UITest() {
+            Board board = new Board();
+            IO io = new IO();
+            this.ui = new UI(board, io);
```
Comment (cdemyanovich) [inline Tests/UITest.cs]: I like that you're using as few instance variables as possible.

File: TicTacToe/TicTacToe/UI.cs [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+            this.io.Print(Greeting());
+            this.io.Print(Instructions());
+            string board = GameBoard(gameBoard);
+            this.io.Print(BoardHeader(TurnPrompt("X")));
+            this.io.Print(BoardBorder(board));
+        }
+
+        public void PlaceMarker() {
+            string input = this.io.GetInput();
+            int move = Int32.Parse(input);
```
Comment (cdemyanovich) [inline TicTacToe/TicTacToe/UI.cs]: Eventually, you'll want to consider how this can fail and handle the various cases gracefully.

Comment (tommcgee) [reply TicTacToe/TicTacToe/UI.cs]: +1 to this

**Pull request merged**